jangjuseong/db/migrations/20230119053017_users.sql

```diff
@@ -0,0 +1,13 @@
+-- migrate:up
+CREATE TABLE users (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    email VARCHAR(200) NOT NULL,
+    password VARCHAR(200) NOT NULL,
+    profile_image VARCHAR(1000) NULL,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+);
```

Comment:
users 테이블에 대한 schema도 잘 설계해주셨네요!

우리가 일반적으로 사용하는 많은 웹사이트를 생각해보면, email이나 유저의 id (계정)에 대해 중복 체크를 확인하는 부분이 있습니다.
클라이언트 (Front-end)단에서 정규표현식을 통해 충분히 처리를 할 수 있지만, 클라이언트를 거치지 않고 외부에서 강압적으로 데이터를 주입하는 경우도 분명 존재할텐데요.

그럴 경우를 방지하여 유저에게 id 이외의 고유한 정보를 받는 경우 해당 컬럼에 대해 unique키 속성을 추가하여 중복되는 데이터가 추가되는 것을 방지할 수 있습니다.

아래와 같이 제약 조건의 이름을 설정할 수도 있습니다.
```suggestion
CREATE TABLE users (
    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
    name VARCHAR(50) NOT NULL,
    email VARCHAR(200) NOT NULL,
    password VARCHAR(200) NOT NULL,
    profile_image VARCHAR(1000) NULL,
    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
    updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
    CONSTRAINT users_email_ukey UNIQUE(email)
);
```